<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XML_project</class>
 <widget class="QMainWindow" name="XML_project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XML Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="graph_button">
          <property name="text">
           <string>Show Graph</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="details_button">
          <property name="text">
           <string>User Details</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="error_check_button">
          <property name="text">
           <string>Check Errors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="error_correct_button">
          <property name="text">
           <string>Correct Errors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="format_button">
          <property name="text">
           <string>Format File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="jason_button">
          <property name="text">
           <string>Show Jason</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_XML_file"/>
   <addaction name="actionSave_Text_as_XML_file"/>
   <addaction name="actionCompress"/>
   <addaction name="actionDecompress"/>
   <addaction name="actionClear"/>
   <addaction name="actionundo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionOpen_XML_file">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Open_icon-icons.com_73704.png</normaloff>:/images/Open_icon-icons.com_73704.png</iconset>
   </property>
   <property name="text">
    <string>Open XML file</string>
   </property>
  </action>
  <action name="actionSave_Text_as_XML_file">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Save Text as XML file</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/10558_green_undo_icon.png</normaloff>:/images/10558_green_undo_icon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/10508_green_redo_icon.png</normaloff>:/images/10508_green_redo_icon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCompress">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/compression.png</normaloff>:/images/compression.png</iconset>
   </property>
   <property name="text">
    <string>Compress File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDecompress">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zip.png</normaloff>:/images/zip.png</iconset>
   </property>
   <property name="text">
    <string>Decompress File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
